README.md

```diff
@@ -198,6 +202,42 @@ IBIS_SCALAR = {
 }
 ```
 
+# Generating Substrait from Adhoc SQL/Ibis
+The CLI tool for generating substrait plans from adhoc SQL queries and Ibis expression
+is located in the `substrait_consumer/tests/adhoc` directory.  The SQL queries should be 
+written using the same TPCH data used in the integration tests.  This tool will generate 
+the substrait plans for each supported producer and run that plan against all supported consumers.
+
+## How to Use
+Copy your SQL query into `substrait_consumer/tests/adhoc/query.sql`.  
+Copy your Ibis expression into `substrait_consumer/tests/adhoc/ibis_expr.py`
```

Comment:
It would probably be easier if we could specify the SQL/ibis paths from the command line?  Do I always need both an ibis and SQL file?  In other words, can I just test SQL?

Reply:
That makes sense.  By default it's going through both right now because i'm using some existing fixtures from the test framework.  I'll create a separate fixture and force the user to specify which producers they want to test out instead.  We'll only check for the ibis file if they specify `IbisProducer`, otherwise we'll check for the SQL file.